docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc

```diff
@@ -88,7 +88,13 @@ automatically if the system is rebooted.
 To confirm that {agent} is installed and running, go to the **Agents** tab in
 {fleet}. 
 
-If you run into problems, see <<fleet-troubleshooting>>.
+If you run into problems:
+
+* Check the {agent} logs. If you use the default policy, agent logs and metrics
```

Comment:
Is lowercase `agent` correct here?

Reply:
Nevermind. I've answered my own question after reading through some of your other PRs.